### PR TITLE
[Chore] update docs to address issue #54

### PR DESCRIPTION
**Pull request by 8ball030**
This Pr is to address concerns raised in;

https://github.com/valory-xyz/open-aea/issues/54

1. Added HTTP demo
2. Added in Apply environment demo
3. Added in vscode launch.json

### CODE REVIEW

**Review by dvilelaf — APPROVED** [review]
LGTM

File: docs/aev_echo_demo.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Configuring with Environment Variables
+The purpose of this demonstration is to show the Open-AEA framework can dynamically configure agents from environment variables.
```
Comment (DavidMinarsch) [inline docs/aev_echo_demo.md:2]:
```suggestion
# Configuring with Environment Variables

The purpose of this demonstration is to show the `open-aea` framework can dynamically configure agents from environment variables.
```

File: docs/aev_echo_demo.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+A full break down of the development flow is covered within the <a href="../quickstart/">Development Quickstart</a>.
+
+It is highly recommended that developers begin by following the quick start!
+
+After you have followed the quick start, create a <a href="../http_echo_demo/">HTTP Echo Agent</a>.
```
Comment (DavidMinarsch) [inline docs/aev_echo_demo.md:8]:
```suggestion
After you have followed the quick start, create an <a href="../http_echo_demo/">HTTP Echo Agent</a>. 
```

File: docs/aev_echo_demo.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+A full break down of the development flow is covered within the <a href="../quickstart/">Development Quickstart</a>.
+
+It is highly recommended that developers begin by following the quick start!
+
+After you have followed the quick start, create a <a href="../http_echo_demo/">HTTP Echo Agent</a>. 
+
+
+Notice, that the configuration of the AEA is as so;
+
+    (open-aea) ~/D/c/v/o/http_echo> cat aea-config.yaml
```
Comment (DavidMinarsch) [inline docs/aev_echo_demo.md:13]: Please display this inside "```yaml" and remove the command line or with command line in "```bash" but without `(open-aea) ~/D/c/v/o/http_echo>`

File: docs/development-setup.md [hunk 4]
```diff
@@ -55,3 +55,29 @@ This advice partially overlaps with the previous two sections:
...
 
 - Alternatively, it can help to provide symbolic links within an AEA to align import paths with folder structure. Simply create an empty file with `touch packages` in your AEA project, then create a symbolic link to `ln -s vendor packages`.
+
+
+## VSCode
+For VSCode modify the ```launch.json``` to include the following information:
+
+
+
+    {
```
Comment (DavidMinarsch) [inline docs/development-setup.md:65]: same, we have convention to not indent but use "```"

File: docs/http_echo_demo.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# HTTP Echo Demo
+
+The purpose of this demonstration is to show the Open-AEA framework can be used as a HTTP server.
```
Comment (DavidMinarsch) [inline docs/http_echo_demo.md:3]:
```suggestion
The purpose of this demonstration is to show the `open-aea` framework can be used as a HTTP server. More concretely, an AEA with a http server connection and an appropriate skill can be used as a server.
```

File: mkdocs.yml [hunk 6]
```diff
@@ -28,8 +28,9 @@ nav:
   - Demos:
     - Demos: 'demos.md'
     - Echo Demo: 'echo_demo.md'
-    - Gym example: 'gym-example.md'
```
Comment (DavidMinarsch) [inline mkdocs.yml:31]: Please remove the `gym-example.md` file

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: docs/aev_echo_demo.md (hunk 2 above)
Comment (8ball030) [reply docs/aev_echo_demo.md:9]: I think should be a HTTP Echo Agent.
Not An HTTP Echo Agent

**Review by DavidMinarsch — APPROVED** [review]
lgtm

**Pull request merged**